src/packages/utils.ts

```diff
@@ -31,6 +31,9 @@ const translateRequestOptionsToFetchOptions = (options: any): { url: string, fet
     if (options.url) {
         url += options.url;
     }
+    if (options.qs) {
+        url += options.qs;
```

Comment:
options.qs should be an object - you will need to convert it to a querystring here

Reply:
Handle question mark case here